examples/lock-app/nrfconnect/README.md

```diff
@@ -210,13 +210,15 @@ To use the Docker container for setup, complete the following steps:
     -   _-e RUNAS=\$(id -u)_ is needed to start the container session as the
         current user instead of root.
 
-4.  Check out or update the nRF Connect SDK to the recommended `v1.5.0` version
-    by running the following command in the Docker container:
+4.  Check out or update the nRF Connect SDK to the recommended version.
+    
+    If you would like to check, if the nRF Connect SDK version you currently have is the recommended one, run the following command in the Docker container:
```

Comment:
This should be universal, and i guess it is.

Reply:
It is universal. If you thought that it works only on Docker, because of the `run the following command in the Docker container`, it was added just to make sure that in that specific case user will type commands in the Docker terminal, not in the system console.